manual_constraints.txt

```diff
@@ -1,3 +1,4 @@
 sphinx_autodoc_annotation >= 1.0
 sphinx_book_theme ~= 1.1.2
-quimb == 1.7.3
\ No newline at end of file
+quimb == 1.7.3
+scipy == 1.12.0
```

Comment:
Should be able to remove this after the next pytket release, maybe worth adding a comment.